swig/scripts/build-android-arm64.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+extract_packages() {
+
+  NDK_VERSION=r21d
+  BOOST_VERSION=1_73_0
+  OPENSSL_VERSION=1.1.1g
+
+  NDK_PKG=${PWD}/../../android-ndk-${NDK_VERSION}-darwin-x86_64.zip
+  BOOST_PKG=${PWD}/../../boost_${BOOST_VERSION}.tar.gz
+  OPENSSL_PKG=${PWD}/../../openssl-${OPENSSL_VERSION}.tar.gz
+
+  rm -rf build
+  mkdir build
+
+  pushd build
+
+  echo "Extracting NDK..."
+  unzip -qq ${NDK_PKG}
+  echo "Extracting BOOST..."
+  tar xzf ${BOOST_PKG}
+  echo "Extracting OpenSSL..."
+  tar xzf ${OPENSSL_PKG}
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/manual/aldenml/libtorrent4j/pullRequest?prid=5908750)